target_chains/ethereum/entropy_sdk/solidity/IEntropy.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache 2
```

Comment:
this is copy/paste of the interface from PythRandom.

I've called this IEntropy to match the new name scheme. I'll circle back and rename the other contracts as well (didn't want to intermingle in this one PR).